packages/functional-tests/pages/relier.ts

```diff
@@ -112,10 +113,14 @@ export class RelierPage extends BaseLayout {
     });
   }
 
-  getUrl() {
-    return this.page
-      .locator('[data-testid=rp-flow-metrics]')
-      .getAttribute('href');
+  async getUrl() {
+    const expectedPath = `\\/subscriptions\\/products\\/${this.target.subscriptionConfig.product}\\?plan=${this.target.subscriptionConfig.plan}&service=`;
+    const subscribeButton = this.page.locator('[data-testid=rp-flow-metrics]');
+    await expect(subscribeButton).toHaveAttribute(
+      'href',
+      new RegExp(`${expectedPath}(\\w+)`, 'i')
```

Comment:
[non-blocking] Might be worthwhile to just make the regex one thing, rather than making it a string, then concatenating it here.

```
const expectedPathRegExp = new RegExp(`\\/subscriptions\\/products\\/${this.target.subscriptionConfig.product}\\?plan=${this.target.subscriptionConfig.plan}&service=(\\w+)`, 'i');
    const subscribeButton = this.page.locator('[data-testid=rp-flow-metrics]');
    await expect(subscribeButton).toHaveAttribute(
      'href',
      expectedPathRegExp
```